vendor/opam/src/core/dune

```diff
@@ -0,0 +1,45 @@
+(library
+  (name        opam_core)
+  (synopsis    "OCaml Package Manager core internal stdlib")
+  (libraries   dune_re unix bigarray sha uutf
+               (select opamACL.ml from
+                       (opam-core.libacl -> opamACL.libacl.ml)
+                       (                 -> opamACL.dummy.ml))
+               (select opamStubs.ml from
+                       (opam-core.stubs -> opamStubs.win32.ml)
+                       (                -> opamStubs.dummy.ml)))
+  (flags       (:standard
+               (:include ../ocaml-flags-standard.sexp)))
```

Comment:
`:include` won't work for bootstrapping so we can just drop it. We disable warnings for vendored sources anyway.